GAE/src/org/waterforpeople/mapping/dataexport/SurveySummaryExporter.java

```diff
@@ -290,8 +291,8 @@ private void writeHeader(PrintWriter pw, boolean isRolledUp) {
         return dtoList;
     }
 
-    protected List<QuestionDto> parseQuestions(String response)
-            throws Exception {
+    
+    protected List<QuestionDto> parseQuestions(String response) throws Exception {
```

Comment:
I think we should replace this parsing with the simpler Jackson deserialiser, like in https://github.com/akvo/akvo-flow/blob/fb7a5b7928047033dc8780a31abd025441076f69/GAE/src/org/waterforpeople/mapping/dataexport/service/BulkDataServiceClient.java#L1396-L1401

That way we don't have to keep adding properties later on if new ones appear.